src/CSParametersRegistry.sol

```diff
@@ -560,6 +560,7 @@ contract CSParametersRegistry is
         uint256 lifetime,
         uint256 threshold
     ) internal pure {
-        if (lifetime < threshold || lifetime < 1) revert InvalidStrikesParams();
+        if (threshold == 0) revert InvalidStrikesParams();
+        if (lifetime == 0) revert InvalidStrikesParams();
```

Comment:
```suggestion
        if (threshold == 0 || lifetime == 0) revert InvalidStrikesParams();
```

Reply:
In general, separate branches are cheaper in solidity. But, it seems that both conditions will be checked most of the time anyway, so I think we can combine them.